tests/checkers/unittest_python3.py

```diff
@@ -1153,3 +1153,37 @@ def next(cls):  #@
         message = testutils.Message("next-method-defined", node=node)
         with self.assertAddsMessages(message):
             self.checker.visit_functiondef(node)
+
+    def test_old_style_super(self):
```

Comment:
These tests should have went in `functional` instead as this test file is for the Python 3 porting checks.